algebird-core/src/main/scala/com/twitter/algebird/SummingCache.scala

```diff
@@ -76,3 +76,18 @@ class SummingCache[K, V] private (capacity: Int)(implicit sgv: Semigroup[V])
       }
   }).asScala
 }
+
+object SummingWithHitsCache {
+  def apply[K, V: Semigroup](cap: Int): SummingWithHitsCache[K, V] = new SummingWithHitsCache[K, V](cap)
+}
+/**
+ * A SummingCache that also tracks the number of key hits
+ */
+class SummingWithHitsCache[K, V] private (capacity: Int)(implicit sgv: Semigroup[V])
```

Comment:
Updated.
Just out of curiosity, what's the original reason to make the constructor private?